DEVELOPMENT_GUIDE.md

```diff
@@ -116,30 +95,116 @@ This step is optional and will use the specified version of
 aws-sam-transformer from PyPi by default.
 
 
-``cd ~/projects (cd into the directory where you usually place projects)``
+```sh
+$ cd ~/projects  # cd into the directory where you usually place projects
```

Comment:
I don't think we need to hand-hold this. Anyone who has used Git will understand the basic functioning of `git clone`. Otherwise should we explain what `cd` does, and how to start a shell, SSH versus HTTPS?

Reply:
good point, updated